src/test/java/com/olgaboiar/mint/ServerTest.java

```diff
@@ -1,39 +1,41 @@
-
-import java.io.*;
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
-class ServerTest {
-    String serverTestRoutesPath = "/testRoutes.yaml";
-    ILogger logger = new MockFileLogger();
-
-    @Test
-    public void serverAnswersSimpleGet() throws IOException {
-        String requestLine = "GET /test HTTP/1.1\nHost: 0.0.0.0:5000\n\nsome_body";
-        StringReader stringReader = new StringReader(requestLine);
-        BufferedReader bufferedReader = new BufferedReader(stringReader);
-        StringWriter stringWriter = new StringWriter();
-        String currentDate = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now());
-        PrintWriter printWriter = new PrintWriter(stringWriter);
-        MockServerConnection serverConnection = new MockServerConnection(printWriter, bufferedReader);
-        Server server = new Server(serverConnection, logger, serverTestRoutesPath);
-        server.start();
-        server.run();
-        String actual = stringWriter.toString();
-        String expected =
-                "HTTP/1.1 200 OK\r\n" +
-                "Date: " + currentDate +
-                "\n\n";
-
-        assertEquals(expected, actual);
-    }
-}
+//package com.olgaboiar.mint;
+//
+//import com.olgaboiar.mint.loggers.ILogger;
+//import com.olgaboiar.mint.loggers.MockFileLogger;
+//import org.junit.jupiter.api.*;
+//
+//import java.io.*;
+//import java.time.ZonedDateTime;
+//import java.time.format.DateTimeFormatter;
+//
+//import static org.junit.jupiter.api.Assertions.assertEquals;
+//
+//
+//@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+//class ServerTest {
+//    String serverTestRoutesPath = "/testRoutes.yaml";
+//    ILogger logger = new MockFileLogger();
+//
+//    @Test
+//    public void serverAnswersSimpleGet() throws IOException {
+//        String requestLine = "GET /test HTTP/1.1\nHost: 0.0.0.0:5000\n\nsome_body";
+//        StringReader stringReader = new StringReader(requestLine);
+//        BufferedReader bufferedReader = new BufferedReader(stringReader);
+//        StringWriter stringWriter = new StringWriter();
+//        String currentDate = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now());
+//        PrintWriter printWriter = new PrintWriter(stringWriter);
+//        MockServerConnection serverConnection = new MockServerConnection(printWriter, bufferedReader);
+//        Server server = new Server(serverConnection, logger, serverTestRoutesPath);
+//        server.start();
+//        server.run();
+//        String actual = stringWriter.toString();
+//        String expected =
+//                "HTTP/1.1 200 OK\r\n" +
+//                "Date: " + currentDate +
+//                "\r\n" +
+//                "Content-Type: null" +
+//                "\n\n";
+//
+//        assertEquals(expected, actual);
+//    }
+//}
```

Comment:
🤔Did something in this PR break the simple get functionality? I'd be happy to pair and get this passing if you'd like.

Reply:
Seems like it was commented out by mistake. It is passing.
https://github.com/olgaboiar/MintHTTPServer/pull/14/commits/c0efc1f83f9ec7224a33fac1e4214988aa673f68